### ISSUE TITLE
pyparsing deprecated attribute

### ISSUE THREAD

**Issue by dpsi** [vesposito/easyucs#21]
I noticed while using easyucs for the first time ever that it would not run:
```
2021-12-20 15:25:02,465 :: INFO :: 10.96.111.27 :: device.py :: in connect :: Trying to connect to UCS System: 10.96.111.27
2021-12-20 15:25:03,429 :: ERROR :: 10.96.111.27 :: device.py :: in connect :: Error while trying to connect to UCS System: 10.96.111.27: module 'pyparsing' has no attribute 'operatorPrecedence'
2021-12-20 15:25:03,431 :: ERROR :: 10.96.111.27 :: easyucs.py :: in init_process :: Impossible to connect to UCS device
```
When I installed the requirements with pip it automatically picked the lastest version of pyparser.

As a workaround I added `pyparsing<3.0.0` to the requirements.txt and then easyucs worked.

**Comment by vesposito** [comment]
Thank you for this issue report. This is due to pyparsing having deprecated operatorPrecedence in 3.x releases, and the UCS SDKs we are using in EasyUCS are still using this deprecated method.
I will raise an issue to the corresponding UCS SDKs to get this fixed. In the meantime, workaround is to use a version of pyparsing lower than 3.0.0, as you did.

**Comment by vesposito** [comment]
This should now be fixed with the release of ucsmsdk 0.9.13: https://github.com/CiscoUcs/ucsmsdk/releases/tag/v0.9.13
